setup.py

```diff
@@ -46,6 +46,7 @@ def desc():
     platforms="any",
     install_requires=[
         "apispec[yaml]>=6.0.0, <7",
+        "authlib>=1.2.1",
```

Comment:
Authlib needs to be an extra requirement

Reply:
I just removed this from `install_requires` - I saw that authlib is already included in the `oauth` section of `extras_require`.